superdesk/publish/formatters/ninjs_formatter.py

```diff
@@ -173,7 +173,13 @@ def _transform_to_ninjs(self, article, subscriber, recursive=True):
         if extra_items:
             ninjs.setdefault(EXTRA_ITEMS, {}).update(extra_items)
 
-        if article.get(EMBARGO):
+        if article.get('embargoed'):
+            try:
+                ninjs['embargoed'] = article['embargoed'].isoformat()
+            except AttributeError:
```

Comment:
In which case the `isoformat` attribute may be missing? If it's a datetime, should not be always present?

Reply:
just in case when it gets ingested and it might not parse the format properly

Reply:
@petrjasek in this case maybe something should notify the user, a `logger.warning` for instance? That would indicate that something is wrong with either the file format or the parser.

Reply:
good point, will update